### ISSUE TITLE
macOS Ventura can't build

### ISSUE THREAD

**Issue by Darkspeed294** [corpnewt/ProperTree#135]
Hello!

right now I am rocking Ventura and it runs ok. I got internet, bluetooth, sound and graphics acceleration, so I am good.
But unfortunately I cannot get proper tree to work to edit plists. I installed 3 versions of python:
-3.9.6
-3.10.5
-python3??
![Screenshot 2022-06-13 at 03 23 24](https://user-images.githubusercontent.com/84881650/173263381-ee00cd09-eca4-4874-a492-bef75a7b8a42.png)

building the app with option 2-4 gives me this error:
![Screenshot 2022-06-13 at 03 24 47](https://user-images.githubusercontent.com/84881650/173263493-1f5ab48e-3893-4f43-9dfe-878f91537562.png)

and the 1st option gives me this:
![Screenshot 2022-06-13 at 03 26 57](https://user-images.githubusercontent.com/84881650/173263664-c81dadd1-81d9-4a32-9c18-7eb75cdb52e5.png)

Last version of macOS I got it working was 12.4, and there it could build with 3.10.5.

**Comment by Darkspeed294** [comment]
![Screenshot 2022-06-13 at 03 22 14](https://user-images.githubusercontent.com/84881650/173263906-b0d0f428-dcaf-4d7d-82dd-15cf9876a2e3.png)
error log from the buildapp.command

**Comment by corpnewt** [comment]
Pretty sure python is borked on Ventura - no support for that currently though as it's in the first dev beta.

**Comment by Darkspeed294** [comment]
yeah I expected something like that. I guess I will have to dig up and hack another old laptop.

**Comment by corpnewt** [comment]
Seems like some users have reported that using the non-universal Python 3.9 builds from python.org work.  I have not personally tested that though.  Might be worth looking into though.

-CorpNewt

**Comment by Darkspeed294** [comment]
Hueyyyyyy nice, it worked!
I downloaded [3.9.13 Intel version](https://www.python.org/ftp/python/3.9.13/python-3.9.13-macosx10.9.pkg) and builded the app, and it shows me the correct UI now! 

thanks CorpNewt!

**Comment by Darkspeed294** [comment]
![Screenshot 2022-06-14 at 04 50 17](https://user-images.githubusercontent.com/84881650/173483097-13bc4dd0-6cab-48c1-9e52-539619408b33.png)

**Comment by jake99x3** [comment]
Can Confirm using 3.9.13 Intel Version fixed any issues I was having with ProperTree on Ventura

**Comment by notAperson535** [comment]
Same, 3.9.13 fixed it for me too.
